external/codecs/Make.defs

```diff
@@ -15,7 +15,7 @@
 # language governing permissions and limitations under the License.
 #
 ###########################################################################
-# apps/netutils/ftpc/Make.defs
+# apps/netutils/codecs/Make.defs
```

Comment:
apps/netutils/codecs/Make.defs
==>
external/codecs/Make.defs

Reply:
I changed all wrong file pathes in Make.defs in the external folder